.github/workflows/js.yml

```diff
@@ -94,7 +94,7 @@ jobs:
 
   windows:
     name: AMD64 Windows 2019 NodeJS ${{ matrix.node }}
-    runs-on: windows-latest
+    runs-on: windows-2019
```

Comment:
@trxcllnt @domoritz Does JS care if running on Windows 2019 or 2022?  If it doesn't we could instead just update the build name.

Reply:
I don't think it should matter at all.